debian/build_virtualenv

```diff
@@ -33,11 +33,13 @@ esac
 # Use --builtin-venv to use the better `venv` module from CPython 3.4+ rather
 # than the 2/3 compatible `virtualenv`.
 
+# Pin pip to 20.3.4 to fix breakage in 21.0 on py3.5 (xenial)
+
```

Comment:
Might be worth adding a note to say we can remove the new line after we drop xenial?

Reply:
I'd leave it as is for now, if that's OK with you. The comment is there to ensure we pick it up when going through the post-Py3.5 purge checklist in https://github.com/matrix-org/synapse/issues/9131, specifically the "Grep source and docs for anything referencing Python 3.5, Stretch, or Xenial and excise it" bit :)